webapi/src/main/scala/org/knora/webapi/messages/util/search/SparqlTransformer.scala

```diff
@@ -5,9 +5,10 @@
 
 package org.knora.webapi.messages.util.search
 
+import zio.Task
+import zio.UIO
 import zio.ZIO
```

Comment:
```suggestion
import zio._
```
includes the next line, but GitHub doesn't let me annotate that